docs/introduction/uml-diagram.rst

```diff
@@ -0,0 +1,24 @@
+UML Diagrams
+============
+
+This section contains UML diagrams for resources per components.
+
+.. note::
+
+    These are the underlying data models and this shows the relationships between the resources,
+    but not all attributes are the exact same as in the API.
+
+Objects API
+-----------
+.. uml_images::
+    :apps: core
```

Comment:
Could you exclude the `Service` model? This is more of an internal thing that is not really interesting from an API perspective
<img width="600" height="149" alt="afbeelding" src="https://github.com/user-attachments/assets/32cb2544-eed2-47e8-b07d-f6d40d41ea13" />